svc-bie-kafka/src/main/resources/application-prod.yaml

```diff
@@ -4,10 +4,10 @@ spring:
     properties:
       schema.registry.url: "${BIE_KAFKA_PLACEHOLDERS_BROKERS:https://schemaregistry.prod.bip.va.gov:443}"
       schema.registry.ssl.protocol: SSL
-      schema.registry.ssl.keystore.location: "file:${KEYSTORE_FILE}"
+      schema.registry.ssl.keystore.location: "${KEYSTORE_FILE}"
       schema.registry.ssl.keystore.password: "${BIE_KAFKA_KEYSTORE_PASSWORD}"
       schema.registry.ssl.keystore.type: "PKCS12"
-      schema.registry.ssl.truststore.location: "file:${TRUSTSTORE_FILE}"
+      schema.registry.ssl.truststore.location: "${TRUSTSTORE_FILE}"
       schema.registry.ssl.truststore.password: "${BIE_KAFKA_TRUSTSTORE_PASSWORD}"
       schema.registry.ssl.truststore.type: "PKCS12"
```

Comment:
This would be more readable as:
```
schema:
  registry:
    url: "${BIE_KAFKA_PLACEHOLDERS_BROKERS:https://schemaregistry.prod.bip.va.gov:443}"
    ssl:
      protocol: SSL
      keystore.location: "${KEYSTORE_FILE}"
      keystore.password: "${BIE_KAFKA_KEYSTORE_PASSWORD}"
      keystore.type: "PKCS12"
...
```

Reply:
agreed, let me update that.